src/Raven.Server/Json/JsonDeserialization.cs

```diff
@@ -94,9 +94,6 @@ private static Expression GetValue(PropertyInfo propertyInfo, ParameterExpressio
                 type == typeof(long) ||
                 type == typeof(int) ||
                 type == typeof(double) ||
-                type == typeof(DateTime) ||
```

Comment:
We'll add it once we'll need it. It should probably read a long value and convert it to UTC DateTime. But for now it made the code simple to have the ticks directly in the status object.